File: C_Programing/Competitive/Problems_Solving/ASSIGNMENT_6/Q3.c
//WAP to find Factorial of given number 



#include<stdio.h>


    int Factorial(int iNo)
{

    if(iNo < 0)
    {
        iNo = -iNo;    // Negative number positive करा
    }

    int iCnt = 0;
    int iFact = 1;

    for(iCnt = 1; iCnt <=iNo; iCnt++)
    {
        iFact = iFact * iCnt;
    }
   
    return iFact;
}


int main()
{
    int iValue = 0,iRet=0;

    printf("Enter number   : ");
    scanf("%d",&iValue);

iRet= Factorial(iValue);
printf("Factorial of number is %d",iRet);

    return 0;
}

File: java_Programing/Competitive/Problems_Solving/ASSIGNMENT_21/Q3.java
//WAP to display all factors of a given no .
import java.util.Scanner;

class Q3
 {
   public static void main(String A[] )
    {
      int iValue=0;
      
        Scanner sobj=new Scanner(System.in);
      
        System.out.println("Enter Number :");
        iValue=sobj.nextInt();


       LogicXx  lobj=new LogicXx();

       lobj.Factorial(iValue);
    }   
 }

 class LogicXx
 {
   
    void Factorial(int iNo)
      
    {
        int iCnt=0;

        for(iCnt=1;iCnt<=iNo;iCnt++)
        {
           if(iNo %iCnt==0)
            {
             System.err.println(iCnt);  
            }
           
         }
    }
}  
